config/settings.yml

```diff
@@ -3,6 +3,16 @@ features:
   reorder_pages: true
   draft_live_versioning: false
 
+forms_api:
+  # Authentication key to authenticate forms-runner to forms-api
+  auth_key: changeme
```

Comment:
Just wondering if it's safer to not have a default in the production config just to avoid it accidentally being used if there's some other accident in setting the necessary ENV var override. Feels like it might be safer to not have a default and add a check that a value must be provided?

Reply:
Agreed, I've created a trello card to take a look at forms-api during firebreak and see if we can customise this further. 
I just wanted to replace like for like at the moment.

Reply:
Ok cool, the recent deployment failure of this PR because it was merged ahead of the corresponding forms-deploy PR is an example where defaults potentially cause problems. forms-admin deployed without the env var overrides and used the defaults in settings.yml which then broke the environment (my preference would be to fail to deploy and the existing version keep running). Its all fixed and no big deal but shows it'll be a worthwhile firebreak activity to investigate removing the defaults 👍